Sources/VaporJWT/ConfigError.swift

```diff
@@ -0,0 +1,26 @@
+/// Errors that can take place during the config
+/// of required JWT entities such as the signer
+public struct ConfigError: Swift.Error {
+    var code: Int
```

Comment:
I can get on board with that, I think maybe the initializer should also be private then and force users through the publicized conveniences. Having the initializer public w/ properties not matching feels a little odd if we're assuming users shouldn't really be able to access or verify their custom initializers. We can always make both public later should it come up